matchbox_socket/Cargo.toml

```diff
@@ -11,6 +11,7 @@ repository = "https://github.com/johanhelsing/matchbox"
 
 [features]
 ggrs-socket = ["dep:bincode", "dep:ggrs"]
+bevy-plugin = ["dep:bevy"]
```

Comment:
Would it perhaps be better to put the bevy plugin in its own crate (which could have a re-export of matchbox_socket)?

We're also depending on bevy 0.9 here. I'd hoped to have it be compatible with 0.10 for the jam. If necessary, we could use my 0.10 branch of bevy_ggrs (just for the bevy_ggrs example #130 ).

Reply:
I think I'd want to keep it consistent with what we do for `ggrs_socket`, but have no strong opinion on whether it's its own crate or not. My inclination would be to stick with keeping it as a feature until there is a compelling reason to do otherwise

Reply:
The most compelling reason as I see it, is that we don't need to release a new version of the socket each time bevy releases.

The reason `ggrs_socket` is a feature, is that you can't implement traits on foreign types. The alternative would be to ask gschup if we could move it to ggrs, but currently we're releasing more frequently than ggrs is.

Reply:
Also, if we want server support in the bevy plugin, perhaps it makes sense that it's a the bevy plugin crate that has an optional feature dependency on the signaling crate from #135 ?

Reply:
I'm fairly convinced by the version decoupling argument, and having it offer `matchbox_signalling` wouldn't hurt either

Will move it over to it's own package. What's your thoughts on re-exporting `matchbox_socket` via it? I would be tempted to. If so, do we give it a `ggrs` feature too?

Reply:
Never mind, have had a look at what other crates do, have re-exported `matchbox_signalling` but haven't added a ggrs feature as they can include that themselves if needs be